notebooks/book2/29/kf_parallel.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.          ax.legend(ncol=2)
</pre><p>ncol=2 - Nice trick:)</p><p>But you don't need if else, I think it's fine to use latexify settings for notebook.</p><p> </p><p>We use <code>if is_latexify_enabled()</code> in case it is much necessary (say nootebook's figure looks terrible) otherwise we avoid to decrease complexity of code. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>